nowcasting_dataset/data_sources/topographic/topographic_data_source.py

```diff
@@ -21,6 +22,7 @@
 # data = out.read(masked=True)
 # print(np.mean(data))
 # print(np.std(data))
+# TODO: Issue #321:  Remove these normalisation variables?
```

Comment:
they are already in Dataloader, so feel free to to drop them here, or do this in a separate PR

Reply:
Cool, yeah, thanks for clarifying.  Yeah, let's drop these as a separate PR (as part of #321)

Reply:
Nice, thanks, done!